core/station/impl/src/core/init.rs

```diff
@@ -145,26 +144,27 @@ lazy_static! {
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
             Resource::Account(AccountResourceAction::Read(ResourceId::Any)),
         ),
-        // create, change, call, and read external canister
+        // external canisters
         (
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
-            Resource::ExternalCanister(ExternalCanisterResourceAction::Create),
+            Resource::ExternalCanister(ExternalCanisterResourceAction::List),
         ),
         (
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
-            Resource::ExternalCanister(ExternalCanisterResourceAction::Change(ExternalCanisterId::Any)),
+            Resource::ExternalCanister(ExternalCanisterResourceAction::Read(ExternalCanisterId::Any)),
         ),
         (
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
-            Resource::ExternalCanister(ExternalCanisterResourceAction::Call(CallExternalCanisterResourceTarget {
```

Comment:
Yes, this way we are consistent with the asset transfers, which admins don't get the permission to transfer from all accounts by default, they would need to add this by their choice.

Reply:
> Yes, this way we are consistent with the asset transfers

But admins can still change the WASM so they can still do essentially anything from the security perspective.

Reply:
that's correct, security wise this changes nothing, it's just to provide a consistent UX and so that users are aware that they are adding the permission and we are not making that decision by default on their behalf.

Reply:
> we are not making that decision by default on their behalf

but then they should also grant the Change permission explicitly